src/connector.cpp

```diff
@@ -717,6 +717,7 @@ bool HostConnector::setJackPorts(const std::array<std::string, 2>& capture, cons
         hostDisconnectAllBlockOutputs(0, lastBlock);
     }
 
+#if MONITOR_AUDIO_LEVELS == 1
```

Comment:
there is no need to compare to 1 for macros, anything != 0 passes the if check and looks cleaner